pkg/controllers/vscr/vdbverify_reconciler.go

```diff
@@ -78,23 +72,28 @@ func (s *VDBVerifyReconciler) Reconcile(ctx context.Context, _ *ctrl.Request) (c
 // vclusterops deployment
 func (s *VDBVerifyReconciler) checkVersionAndDeploymentType(ctx context.Context) error {
 	if !vmeta.UseVClusterOps(s.Vdb.Annotations) {
-		return s.updateScrutinizeReadyCondition(ctx, metav1.ConditionFalse, vclusterOpsDisabled)
+		s.VRec.Eventf(s.Vscr, corev1.EventTypeWarning, events.VclusterOpsDisabled,
+			"The VerticaDB named '%s' has vclusterops disabled", s.Vdb.Name)
+		return s.updateScrutinizeReadyCondition(ctx, metav1.ConditionFalse, events.VclusterOpsDisabled)
 	}
 	vinf, ok := s.Vdb.MakeVersionInfo()
 	if !ok {
 		// if we don't find the server version in the vdb, we assume the vdb
 		// is not ready for scrutinize and exit
-		s.Log.Info("Vertica version is not available in the vdb")
-		return s.updateScrutinizeReadyCondition(ctx, metav1.ConditionFalse, verticaVersionNotFound)
+		s.VRec.Eventf(s.Vscr, corev1.EventTypeWarning, events.VerticaVersionNotFound,
+			"The VerticaDB named '%s' does not have the version annotation set", s.Vdb.Name)
+		return s.updateScrutinizeReadyCondition(ctx, metav1.ConditionFalse, events.VerticaVersionNotFound)
 	}
 
 	if vinf.IsOlder(v1.VcluseropsAsDefaultDeploymentMethodMinVersion) {
 		ver, _ := s.Vdb.GetVerticaVersionStr()
-		s.VRec.Eventf(s.Vscr, corev1.EventTypeWarning, events.UnsupportedVerticaVersion,
+		s.VRec.Eventf(s.Vscr, corev1.EventTypeWarning, events.VclusterOpsScrutinizeNotSupported,
 			"The server version %s does not have scrutinize support through vclusterOps", ver)
-		return s.updateScrutinizeReadyCondition(ctx, metav1.ConditionFalse, vclusterOpsScrutinizeNotSupported)
+		return s.updateScrutinizeReadyCondition(ctx, metav1.ConditionFalse, events.VclusterOpsScrutinizeNotSupported)
 	}
-	return s.updateScrutinizeReadyCondition(ctx, metav1.ConditionTrue, verticaDBSetForScrutinize)
+	s.VRec.Eventf(s.Vscr, corev1.EventTypeNormal, events.VerticaDBSetForScrutinize,
```

Comment:
This is a normal condition, so probably doesn't warrant its own event. We should use events for errors we want to surface to the callers or for long running operations.